lib/Modules/ComputeHooks/ComputeHooks.psm1

```diff
@@ -430,6 +472,11 @@ function Get-FreeRDPContext {
     if (!$ctx.Count) {
         return @{}
     }
+    $cfg = Get-JujuCharmConfig
+    $rdpBaseUrl = $cfg['html5-console-url']
```

Comment:
Any reason why we would want to overwrite the value coming in from the relation?